odh-dashboard/base/kustomization.yaml

```diff
@@ -24,7 +24,7 @@ resources:
 images:
 - name: odh-dashboard
   newName: quay.io/opendatahub/odh-dashboard
-  digest: sha256:08e85d8b4b4ef255ace46c6587e528a533da723e56c5faa63170dcbc2c2bf71c
+  newTag: main
```

Comment:
Need to update this to the `nightly` build once it collects the last of the changes from the release and builds over night.